src/Compilers/Core/MSBuildTask/Directory.Build.targets

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
It would be useful to know how much of these Directory files are just extracted from the single project we had before

Reply:
Virtually the entire file is a cut / paste from the original project file.